src/languages/python/emit.ts

```diff
@@ -235,6 +235,8 @@ export default function emitProgram(
         }
         case "Prefix":
           return [e.name, emit(e.arg, prec)];
+        case "Set":
+          return ["{", joinNodes(",", e.exprs), "}"];
```

Comment:
This is a bitset which can only hold ordinal types with no more than 2^16 elements. Should probably use a hashset fro simplicity, otherwise targetType flooding ala bigint in Js might be needed.

Reply:
I was not aware of that, how would you specify a hashset rather than a bitset?

Reply:
> I was not aware of that, how would you specify a hashset rather than a bitset?

`["[", joinNodes(",", e.exprs), "]",".","toSet"]` plus `Set: "sets"` row in `addNimImports`. A bit hacky, but this is how tables are emitted rn.

Reply:
Ah, this code is for Python. I can go ahead and add that set notation to Nim though.